try impl #35

lgc2333 opened the pull request:
我并不知道这样的做法是否优雅，如果不喜欢请拒绝
resolve #35

lgc2333 committed:
don't hide smtc when external control disabled

lgc2333 committed:
Merge branch 'MATRIX-feather:ruleset' into ruleset

lgc2333 committed:
Merge branch 'MATRIX-feather:ruleset' into ruleset

MATRIX-feather commented on LLin.OSIntegrations/Windows/WindowsMediaIntegration.cs:
```diff
@@ -148,7 +149,8 @@ private void smtcPlaybackRateChangeRequested(SystemMediaTransportControls sender
     {
-        SetPosition?.Invoke(args.RequestedPlaybackPosition.Milliseconds);
+        if (AllowExternalControl)
+            SetPosition?.Invoke(args.RequestedPlaybackPosition.Milliseconds);
```
在 `MediaIntegration#load` 中，LLin 会检查当前情况是否允许执行控制指令，因此我觉得在针对平台的实现中不需要再实行进一步检查

MATRIX-feather replied:
刚发现这个 review 还在 Pending 没发出来，对不起喵🐱

lgc2333 commented on LLin.OSIntegrations/Windows/WindowsMediaIntegration.cs:
```diff
@@ -160,6 +162,8 @@ private void smtcPropertyChanged(SystemMediaTransportControls sender,
                                    SystemMediaTransportControlsButtonPressedEventArgs args)
     {
+        if (!AllowExternalControl)
+            return;
```
此处是否需要修改

MATRIX-feather replied:
也是需要的

lgc2333 committed:
Simplify playback position change handling

Removed unnecessary conditional check for AllowExternalControl in playback position change handling.

MATRIX-feather commented:
thx :heart:
